### PR TITLE
etcdctlv3: interactive_v3 compaction and timetolive command update

### PR DESCRIPTION
**Pull request by sinsharat**
1. added note for compaction to give info about how to get revision in order to perform compaction operation.
2. Added timetolive operation command and its example.

### CODE REVIEW

File: Documentation/dev-guide/interacting_v3.md [hunk 1]
```diff
@@ -352,6 +352,13 @@ $ etcdctl get --rev=4 foo
 Error:  rpc error: code = 11 desc = etcdserver: mvcc: required revision has been compacted
 ```
 
+Note: The current revision of etcd server can be found using get command on any key (existent or non-existent) in json format. Example is shown below for mykey which does not exist in etcd server :
```
Comment (xiang90) [inline Documentation/dev-guide/interacting_v3.md]: remove the space between `etcd server` and `:`?

File: Documentation/dev-guide/interacting_v3.md [hunk 2]
```diff
@@ -411,3 +418,40 @@ lease 32695410dcc0ca06 keepalived with TTL(100)
 lease 32695410dcc0ca06 keepalived with TTL(100)
 ...
 ```
+
+## Get information about lease
```
Comment (xiang90) [inline Documentation/dev-guide/interacting_v3.md]: Get lease information

**Comment by xiang90** [comment]
LGTM after fixing the nit.

File: Documentation/dev-guide/interacting_v3.md (hunk 1 above)
Comment (sinsharat) [inline Documentation/dev-guide/interacting_v3.md]: sure will remove it.

File: Documentation/dev-guide/interacting_v3.md (hunk 2 above)
Comment (sinsharat) [inline Documentation/dev-guide/interacting_v3.md]: ok will change the title.

**Comment by sinsharat** [comment]
@xiang90 I have done the changes as per you suggestion.
Thank you!

**Pull request merged**